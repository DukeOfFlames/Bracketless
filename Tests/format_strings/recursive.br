START
#drucke{"The length of the string '"10^6 = '10^6'"' is '#count{"10^6 = '10^6'"}'"}
END