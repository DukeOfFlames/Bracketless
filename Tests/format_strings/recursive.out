"The length of the string "10^6 = 1000000" is 14"
